src/pages/workspace/taxes/WorkspaceTaxesPage.tsx

```diff
@@ -66,34 +76,34 @@ function WorkspaceTaxesPage({policy, route}: WorkspaceTaxesPageProps) {
         [defaultExternalID, foreignTaxDefault, translate],
     );
 
-    const taxesList = useMemo<ListItem[]>(
-        () =>
-            Object.entries(policy?.taxRates?.taxes ?? {})
-                .map(([key, value]) => ({
-                    text: value.name,
-                    alternateText: textForDefault(key),
-                    keyForList: key,
-                    isSelected: !!selectedTaxesIDs.includes(key),
-                    isDisabledCheckbox: key === defaultExternalID,
-                    pendingAction: value.pendingAction,
-                    errors: value.errors,
-                    rightElement: (
-                        <View style={styles.flexRow}>
-                            <Text style={[styles.disabledText, styles.alignSelfCenter]}>
-                                {value.isDisabled ? translate('workspace.common.disabled') : translate('workspace.common.enabled')}
-                            </Text>
-                            <View style={[styles.p1, styles.pl2]}>
-                                <Icon
-                                    src={Expensicons.ArrowRight}
-                                    fill={theme.icon}
-                                />
-                            </View>
+    const taxesList = useMemo<ListItem[]>(() => {
+        if (!policy) {
+            return [];
+        }
+        return Object.entries(policy.taxRates?.taxes ?? {})
+            .map(([key, value]) => ({
+                text: value.name,
+                alternateText: textForDefault(key),
+                keyForList: key,
+                isSelected: !!selectedTaxesIDs.includes(key),
+                isDisabledCheckbox: !canEditTaxRate(policy, key),
+                isDisabled: value.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE,
+                pendingAction: value.pendingAction ?? (Object.keys(value.pendingFields ?? {}).length > 0 ? CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE : null),
```

Comment:
Why do we have check for other pendingFields here? Isn't `value.pendingAction` enough?
```suggestion
                pendingAction: value.pendingAction ?? (Object.keys(value.pendingFields ?? {}).length > 0 ? CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE : null),
```

Reply:
When we update the tax, including toggling it or changing its name or value, we store this information in the `pendingFields` parameter, rather than in the `pendingAction`. This is done to indicate to the user exactly which field is pending an update.

It seems reasonable to assume that if a tax has a pending field update, we should also indicate this to the user on the tax list.